### DISCUSSION TITLE
Confirm before Deleting a Shopping List

### DISCUSSION THREAD

**Discussion by kabukimandelux** [mealie-recipes/mealie#848] [Feature Request]
**Before submitting this feature request I have**
_COMPLETE AND FILL THESE OUT_
- [ x] Reviewed the documentation and verified that my feature request isn't already a feature.
- [x ] Reviewed the existing feature requests and my feature has not been requested.
- [x ] Checked the [tasks taged](https://github.com/hay-kot/mealie/issues?q=is%3Aissue+is%3Aopen+label%3Atask+) issues and verified my feature is not covered

**Please Describe The Problem To Be Solved**
Avoid accidental delete of a shopping list when touching the delete button. The likelihood of this happening is high, especially when ticking of a shopping list on a phone.

**(Optional): Suggest A Solution**
Implement a popup box to confirm delete. This could be mandatory or an option in the settings.

**Additional Information**
- [ ] If the feature is accepted I'm willing to submit a PR to provide this feature

**Comment by hay-kot** [comment]
Will be sure to add this in v1!
